documents-webapp/src/main/webapp/vue-app/documents/components/DocumentsMain.vue

```diff
@@ -5,20 +5,26 @@
     role="main"
     flat>
     <div class="pa-3 white">
-      <documents-header
-        class="py-2"
-        :is-mobile="isMobile" />
-      <documents-body
-        :view-extension="selectedViewExtension"
-        :files="files"
-        :groups-sizes="groupsSizes"
-        :page-size="pageSize"
-        :offset="offset"
-        :limit="limit"
-        :has-more="hasMore"
-        :sort-field="sortField"
-        :ascending="ascending"
-        :loading="loading" />
+      <div v-if="!filesLoad">
+        <documents-header-left
+          class="py-2" />
+        <documents-no-body />
+      </div>
+      <div v-else>
+        <documents-header
+          class="py-2" />
```

Comment:
The isMobile prop has to be passed to the 'documents-header' component it is needed.